doc/articles/toc.yml

```diff
@@ -412,6 +412,8 @@
           items:
             - name: Upgrading from previous releases
```

Comment:
Should this be `Migrating` as well for consistency?

Reply:
Fair point, I'll use upgrading instead in both then.